airflow/dags/create_external_tables/rt_service_alerts_outcomes.yml

```diff
@@ -1,7 +1,11 @@
 operator: operators.ExternalTable
 bucket: gs://rt-parsed
 prefix_bucket: true
-test: true
+
+post_hook: |
+  select distinct step
```

Comment:
**question**: why `select distinct step`? Will this catch if there are issues in other columns? Did you find that `select * limit 1` (which would be my instinct) was bad for some reason?

Reply:
This is just to test that we can parse a particular column and it's not going to return a ton of results even if we query the entire data set.